tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java

```diff
@@ -530,16 +530,16 @@ private void joinWithOneResource(String methodName, boolean close, String resour
             lraClient.closeLRA(lra);
             lraTestService.waitForCallbacks(lra);
             
-            assertTrue(methodName + ": resource should have completed once with no compensations",
-                    1 == lraMetricService.getMetric(LRAMetricType.Completed, lra, ParticipatingTckResource.class.getName()) &&
+            assertTrue(methodName + ": resource should have completed with no compensations",
+                    1 <= lraMetricService.getMetric(LRAMetricType.Completed, lra, ParticipatingTckResource.class.getName()) &&
                     0 == lraMetricService.getMetric(LRAMetricType.Compensated, lra, ParticipatingTckResource.class.getName()));
         } else {
             lraClient.cancelLRA(lra);
             lraTestService.waitForCallbacks(lra);
 
-            assertTrue(methodName + ":: resource should have compensated once with no completions",
```

Comment:
Same comment

Reply:
I see that you have added the deleted line back in again so I will mark this conversation as resolved.